e2e/test_operator_backups/operator_backup_test.go

```diff
@@ -168,6 +172,41 @@ var _ = Describe("Operator Backup", Label("e2e", "pr"), func() {
 						backupConfiguration.EncryptionEnabled = true
 					})
 
+					When("running describe command", func() {
+						var describeOutput string
+
+						BeforeEach(func() {
+							skipRestore = true
+						})
+
+						JustBeforeEach(func() {
+							describeOutput = backup.RunDescribeCommand()
+						})
+
+						// TODO (09harsh): Enable this test when we have the fileLevelEncryption in json parser
+						// here: https://github.com/apple/foundationdb/blob/main/fdbclient/BackupContainer.actor.cpp#L193-L250
+						PIt("should have file level encryption enabled", func() {
+							var describeData map[string]interface{}
+							err := json.Unmarshal([]byte(describeOutput), &describeData)
```

Comment:
If we plan to make further checks on the output of the describe command it would be useful to create a struct that holds all the important fields.